views/index.jade

```diff
@@ -37,14 +37,47 @@ block content
         .form-group#buttons
             button.btn.btn-default(type='button' id='btnGenerate') Generate Docs
             .row
-              a.btn.btn-default(type='button' id='btnDownload') Download
-              a.btn.btn-default(type='button' id='btnPreview' target='_blank') Preview
-              a.btn.btn-default(type='button' id='btnDeploy') Deploy
+              a.btn.btn-default(id='btnDownload') Download
+              a.btn.btn-default(id='btnPreview' target='_blank') Preview
+              a.btn.btn-default(id='btnDeploy' data-toggle='modal', data-target='#deployModal') Deploy
       .progress
         #progress.progress-bar(role='progressbar', aria-valuenow='0', aria-valuemin='0', aria-valuemax='100', style='width:0%')
           span.sr-only
       .logs.pre-scrollable
         h4(style='text-align: center; padding-top: 5px; color: #ffffff') Console Output
         ul#messages
+  #deployModal.modal.fade(role='dialog')
+    .modal-dialog
+      .modal-content
+        .modal-header
+          button.close(type='button', data-dismiss='modal') ×
+          h5.modal-title Github Deployment
+        .modal-body
+          p
+            | In order to enable continuous deployment of Documentation, add the following content to your &nbsp;
+            code .travis.yml
+            | , making necessary changes
+          pre
+            | # Add environment variable following description in http://yaydoc.fossasia.org/#usage
+            | env:
+            |   global:
+            |     - AUTHOR=&lt;author-name&rt;
+            |     - DOCPATH=&lt;doc-path&rt;
+            |     - PROJECTNAME=&lt;project-name&rt;
+            |     - VERSION=&lt;version-number&rt;
+            |     - ENCRYPTION_KEY=&lt;32 character alphanumeric string&rt;
```

Comment:
it is needed in the crypt function. A user need to set this variable to enable github deployment, as this is used in encrypting the personal access token